main.py

```diff
@@ -46,6 +71,26 @@ def submit(driver, id_='driving-licence-submit'):
 
 
 def notify(msg):
+    msg += '\nHead over to https://driverpracticaltest.dvsa.gov.uk/ to book now'
+    if ENABLE_TELEGRAM:
+        send_text = f'https://api.telegram.org/bot{TELEGRAM_BOT_TOKEN}/sendMessage?chat_id={TELEGRAM_CHAT_ID}&parse_mode=Markdown&text=' + msg
+        requests.get(send_text)
```

Comment:
I'm not familiar with the telegram API, but it seems a little odd to send all of these parameters in the query string as a GET request, rather than POSTing the message content to the `sendMessage` endpoint - https://core.telegram.org/bots/api#making-requests seems to suggest that you can do this?

Reply:
Updated to POST although not much of a difference